app/main.py

```diff
@@ -2,0 +1,16 @@
+def copy_file(command: str) -> None:
+    splited = command.split(" ")
+    if len(splited) != 3 or splited[0] != "cp":
+        return None
+
+    original_file = splited[1]
+    copied_file = splited[2]
+
+    if original_file == copied_file:
+        return None
```

Comment:
Similar to the previous comment, returning `None` explicitly is not needed. You can simply use `return` to exit the function.